full-service/src/bin/transaction-signer.rs

```diff
@@ -352,7 +352,11 @@ fn tx_out_belongs_to_account(tx_out: &TxOut, account_view_private_key: &Ristrett
         Ok(k) => k,
     };
     let shared_secret = get_tx_out_shared_secret(account_view_private_key, &tx_out_public_key);
-    tx_out.masked_amount.get_value(&shared_secret).is_ok()
+    tx_out
+        .get_masked_amount()
+        .unwrap()
```

Comment:
unwrap seems wrong here, it appears the function should return false if this fails.

Reply:
This code in particular will be addressed in the following PR #436